server/src/api/resources/payment/payment.restRouter.js

```diff
@@ -0,0 +1,8 @@
+import express from "express"
+import * as paymentController from "./payment.controller"
+
+export const paymentRouter = express.Router()
+
+paymentRouter
+  .route("/")
+  .get(paymentController.get)
```

Comment:
Let’s go ahead and remove the `Payment` resource. It’s kinda redundant to have, our Stripe integration should cover bases here